src/core/runtime/context.h

```diff
@@ -121,13 +130,14 @@ class TaskContext {
  public:
   ReturnValues pack_return_values() const;
 
- private:
+ public:
```

Comment:
Why did you want to expose the private members here?

Reply:
ahh this should be reverted. This is an artifact from when we were using the inline launcher.